dora/core/common/src/main/java/alluxio/util/FeatureUtils.java

```diff
@@ -116,4 +117,23 @@ public static boolean isMasterAuditLoggingEnabled() {
   public static boolean isPageStoreEnabled() {
     return Configuration.get(PropertyKey.WORKER_BLOCK_STORE_TYPE) == BlockStoreType.PAGE;
   }
+
+  /**
+   * Utility to check if running on the new DORA architecture(Alluxio 3.x).
+   *
+   * @return true if the architecture is DORA
+   */
+  public static boolean isDora() {
```

Comment:
After some thoughts I decided no to do this. The reasons are:
1. People do `conf.getBoolean(DORA_ENABLED)` and `Configuration.getBoolean(DORA_ENABLED)`. In order to support both, I need `isDora()` and `isDora(conf)`. And it's easier for the user to call the wrong `isDora()` when he/she should really use `isDora(conf)`. Without this interface, the user is more likely to use the correct conf instance.
2. This `DORA_ENABLED` key is temporary anyways.